packages/vcvrack/vcvrack.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh -e
```

Comment:
> removed the quotes around the args to allow multiple arguments.

Sorry, that is not advisable. `$@` is special. It expands to each passed command line argument quoted separately, which is correct, because it preserves quoted command line args.

Reply:
Wow, thanks, learned something new today! Just reverted that :)